src/twisted/words/test/test_irc.py

```diff
@@ -1634,7 +1644,13 @@ def setUp(self):
 
 
     def check(self, s):
-        self.assertEqual(self.f.getvalue(), s)
+        bufferValue = self.f.getvalue()
+        if isinstance(s, unicode):
+            # Convert bufferValue from bytes
```

Comment:
Try to avoid using comments; instead, put this information into the docstring.  (i.e.: add a docstring explaining this method.)